### DISCUSSION TITLE
Update options

### DISCUSSION THREAD

**Discussion by kl3sk** [easepick/easepick#28] [Q&A]
Hello,

Is there a way to update options passed to `.create` method. I didn't find any `updateOption(s)`  nor `setOption(s)`

For example set `inline` mode on the fly ? (what about plugins options too ?)

Thanks.

**Comment by wakirin** [comment]
Hello,
you can change the options directly in the instance of the created picker.
To enable inline mode, you must change the style manually, because the styles for inline mode are applied when the picker is creating.

Here is an example: https://jsfiddle.net/waki/gev0buys/

For plugins you can do the same, but first you need to get instance of plugin.

https://jsfiddle.net/waki/gu0paskc/

Note: `PluginManager` is not currently documented. I will add a description about it soon.
